packs/mmonit/sensors/events_sensor.yaml

```diff
@@ -0,0 +1,22 @@
+---
+  class_name: "MmonitEventsSensor"
+  entry_point: "events_sensor.py"
+  description: "Sensor which monitors mmonit for events"
+  poll_interval: 10
+  trigger_types:
+    -
+      name: "new_event"
+      description: "Trigger which indicates that a new event has arrived at mmonit"
+      payload_schema:
+        type: "object"
+        properties:
+          host:
+            type: "string"
+          id:
+            type: "integer"
+          event:
```

Comment:
Can you elaborate on this?
The possible values are know, yes. Do you mean passing the whole object? Does that allow for rule filtering based on fields?